datashare-app/src/main/java/org/icij/datashare/mode/CommonMode.java

```diff
@@ -132,6 +132,7 @@ public Injector createChildInjector(Module... modules) {
     protected void configure() {
         bind(PropertiesProvider.class).toInstance(propertiesProvider);
         install(new FactoryModuleBuilder().build(TaskFactory.class));
+        install(new FactoryModuleBuilder().build(org.icij.datashare.asynctasks.TaskFactory.class));
```

Comment:
as `TaskFactory` is having its parent `org.icij.datashare.asynctasks.TaskFactory` (that is empty) we shouldn't install the other : either the parent is known in the class path and the children is defined, either not and there should be exceptions while compiling.

Reply:
it seems that guice is looking for a concrete instance for the parent class `org.icij.datashare.asynctasks.TaskFactory`. So we have to define the parent as a factory.